### ISSUE TITLE
The hint of semantic widget will voice out twice in android

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#60436]
b/159579926

Observe that when using Semantics widget in Flutter on android platform, the hint will voice out twice.

Create Semantics widget as below.

```dart
Semantics(
  label: 'label',
  hint: 'hint',
  onTap: () {},
  button: true,
  child: ...,
```

Turn on VoiceOver / TalkBack, and swipe to focus on the widget.

- On iOS, it will voice out `label - button - hint`.
- On Android, it will voice out `label - hint - button - hint`

**Comment by nscobie** [comment]
_(Unable to reproduce, following up with the customer internally. Asking for any corrections to my repro app, and which Android device and Flutter version they're using.)_

My test setup:

- Flutter: `1.20.0-3.0.pre.68`
- Phone: `Pixel 3a`
- Android: `10`

Focusing on a Text widget with text 'child' under the Semantics widget reads the following for me:

_**label - child - hint - button - double tap to activate**_

Sample code:
```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

// Trying to repro https://github.com/flutter/flutter/issues/60436
class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: Center(
          child: Semantics(
            label: 'label',
            hint: 'hint',
            onTap: () {},
            button: true,
            child: Text('child'),
          ),
        ),
      ),
    );
  }
}
```

**Comment by nscobie** [comment]
Closing as WAI, see discussion with customer on b/159579926.
